cumulusci/tasks/salesforce/package_upload.py

```diff
@@ -59,6 +70,43 @@ def _run_task(self):
             self._set_dependencies()
             self._log_package_upload_success()
 
+    """This functions validates the major version and minor version if passed and sets them in the options dictionary if not passed."""
```

Comment:
Triple-quotes following a declaration in Python are a _docstring_, a built-in help text. I would suggest putting this docstring on line 76, which associates it with the function `_validate_versions()`.